scripts/rollup.config.js

```diff
@@ -0,0 +1,97 @@
+import babel from 'rollup-plugin-babel';
+import commonjs from 'rollup-plugin-commonjs';
+import json from 'rollup-plugin-json';
+import resolve from 'rollup-plugin-node-resolve';
+import uglify from 'rollup-plugin-uglify';
+import pkg from '../package.json';
+
+const date = new Date();
+
+const banner =
+`/**
+ * ${pkg.name}
+ * @version ${pkg.version}
+ * @copyright ${date.getFullYear()} ${pkg.author}
+ * @license ${pkg.license}
+ */`;
+
+export default [
```

Comment:
This block has a lot of repetition - everything outside `output` it looks like - so seems like a good candidate for being more DRY.